### PR TITLE
Add test for options to babel-cli

### PR DESCRIPTION
**Pull request by outsideris**
| Q                        | A 
| ------------------------ | ---
| Patch: Bug Fix?          | N
| Major: Breaking Change?  | N
| Minor: New Feature?      | N
| Deprecations?            | 
| Spec Compliancy?         | 
| Tests Added/Pass?        | Y
| Fixed Tickets            | 
| License                  | MIT
| Doc PR                   | 
| Dependency Changes       | 

It's tests for babel-cli's options, see [the coverrage](https://codecov.io/gh/babel/babel/src/793d5a7528b9bb9fdb37c64f3ac5f3358c059a1c/packages/babel-cli/src/babel-node.js#L40).

### CODE REVIEW

**Comment by outsideris** [comment]
I'm stuck here.

1. Node v4 doesn't support `--inspect`. How can I handle it?
2. The tests were passed in Node v7, but nyc which is for coverage looks like doesn't support `debug` options. 

Also, I tried to add tests for `--d`, `--debug` and `--debug-brk`. But it is hard to make tests for them because these flags are hanging in debug console.

**Comment by xtuc** [comment]
If the test can't run on Node v4 I would skip it, like:

```js
it('foo', function() {
  if (getMajorVersion(process.version) === 4) {
    this.skip();
  }

});
```

**Comment by outsideris** [comment]
I removed a test for a `debug` option, because there is an error with `nyc`.

File: packages/babel-cli/test/index.js [hunk 1]
```diff
@@ -174,7 +174,9 @@ fs.readdirSync(fixtureLoc).forEach(function (binName) {
         opts.inFiles[".babelrc"] = helper.readFile(babelrcLoc);
       }
 
-      it(testName, buildTest(binName, testName, opts));
+      if (testName !== "flag-dashed-inspect" || !process.version.match(/^v[045]/)) {
```
Comment (outsideris) [inline packages/babel-cli/test/index.js]: I want to exclude the test for  `--inspect` in Node v0.12, v4 and v5.
I doesn't like this workaround, but I couldn't find better way.

Comment (existentialism) [inline packages/babel-cli/test/index.js]: We recently landed https://github.com/babel/babel/pull/5765, which allows you to specify a minimum node.js version via `minNodeVersion`

**Comment by existentialism** [comment]
@outsideris sorry for the delay on this, but let's revive this as we landed https://github.com/babel/babel/pull/5785 and should probably get some test coverage around this.

**Comment by xtuc** [comment]
@existentialism I changed the label because you set (by mistake?) the documentation one.

**Comment by outsideris** [comment]
I'm trying to fix it and change the base branch to 7.0. 
However although I specify `minNodeVersion`, it failed in [Node v4](https://travis-ci.org/babel/babel/jobs/241721113)
I missed something? @existentialism

**Comment by jridgewell** [comment]
I think `minNodeVersion` only work in fixture tests, not babel-cli tests.

**Comment by outsideris** [comment]
So, how can I solve it?

**Pull request closed without merging**